src/main/java/com/wcc/platform/service/CmsService.java

```diff
@@ -18,69 +16,86 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
-/** CMS service responsible for simple pages. */
+/**
+ * CMS service responsible for simple pages.
+ */
 @Service
 public class CmsService {
-  private final ObjectMapper objectMapper;
 
-  @Autowired
-  public CmsService(final ObjectMapper objectMapper) {
-    this.objectMapper = objectMapper;
-  }
+    private final ObjectMapper objectMapper;
 
-  /**
-   * Read JSON and convert to Pojo TeamPage.
-   *
-   * @return Pojo TeamPage.
-   */
-  public TeamPage getTeam() {
-    try {
-      final File file = Path.of(FileUtil.getFileUri(TEAM.getFileName())).toFile();
-      return objectMapper.readValue(file, TeamPage.class);
-    } catch (IOException e) {
-      throw new PlatformInternalException(e.getMessage(), e);
+    @Autowired
```

Comment:
I think the project does not have autoformat configured using google-format, because the format should not change. Please have a look in the README, how to setup intelliJ